styles/styles.css

```diff
@@ -6,3 +6,148 @@
  * 
  *
  */
+
+/* main > div {
+  width: 100%!important;
+  margin: 0 auto;
+  max-width: 1260px;
+  padding-left: 70px!important;
+  padding-right: 70px!important;
+} */
+
+body {
+  line-height: 21px;
+}
+
+body table {
+  font-size: 16px;
+  border-spacing: 0;
+}
+
+table, th, td {
+  padding: 9px 10px;
+  border: 1px solid #bdbdbd!important;
+  border-collapse: collapse!important;
+}
+
+table th {
+  background-color: #ededed;
+}
+
+.columns {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.accordion dd div {
+  padding-top: 10px;
+}
+
+.content {
+  display: block;
+  max-width: var(--grid-container-width);
+  margin: 0 auto;
+}
+
+/*
+* title
+*/
+
+.page-title {
+  padding: 10px!important;
```

Comment:
Is there a way we could increase the specificity of the selector so that we could get rid of `!important`

Reply:
The way would be to use another way to handle the title in the pages (remember JIRA ticket MWPW-122847 where the different title authoring solutions are described).
For now I'm re-using an existing Milo block to handle the title. 
I'm suggesting we go we this solution for now as we need to know how the authors would like to write the title in the document (MWPW-122847) to know better how to handle the block.